src/compiler/objective_c_plugin.cc

```diff
@@ -104,6 +110,19 @@ class ObjectiveCGrpcGenerator : public grpc::protobuf::compiler::CodeGenerator {
           return false;
         }
         framework = param[1];
+      } else if (param[0] == "runtime_import_prefix") {
+        if (param.size() != 2) {
+          *error = grpc::string("Format: runtime_import_prefix=dir/");
+          return false;
+        } else if (param[1].empty()) {
+          *error = grpc::string(
```

Comment:
I thought an empty parameter value is equivalent to the parameter not exists in this case, but correct me if I'm wrong.

Reply:
Done, good point.